examples/music-v3.py

```diff
@@ -33,11 +33,12 @@ def __unload(self):
             player.cleanup()
         # Clear the players from Lavalink's internal cache
         self.bot.lavalink.players.clear()
-        self.bot.lavalink.unregister_hook(self._track_hook)

```

Comment:
Also here :^) ... call `self.bot.lavalink.destroy()` to safely disconnect websockets.